library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -443,8 +443,40 @@ public int markRowDeleted(long... ids) {
      * @param ids the IDs of the downloads to remove
      * @return the number of downloads actually removed
      */
-    public int remove(long... ids) {
-        return markRowDeleted(ids);
+    public int removeDownloads(long... ids) {
```

Comment:
in this situation I would refer back to original source to see if there is something we have deleted that gives us more understanding.

Aaaand it doesn't help... http://androidxref.com/4.4.4_r1/xref/frameworks/base/core/java/android/app/DownloadManager.java#markRowDeleted